src/libsync/theme.cpp

```diff
@@ -577,19 +577,14 @@ QString Theme::versionSwitchOutput() const
 
 bool Theme::showVirtualFilesOption() const
 {
-    return enableExperimentalFeatures();
+    return true;
```

Comment:
Supported for historical reasons and prevent refactoring? Or are we going to provide this option in the future?

Reply:
Supported in branding